### DISCUSSION TITLE
Unable to find apt.list conataining https://packages.sury.org/php jammy Release

### DISCUSSION THREAD

**Discussion by G400** [oerdnj/deb.sury.org#2078] [Q&A]
Ubuntu 22.04.3:
apt update throws this error:
Err:14 https://packages.sury.org/php jammy Release
  404  Not Found [IP: 194.242.11.186 443]
  
  I have checked all lists I can think of, but I can't find any where this package is included
  I would appreciate help

**Comment by todeveni** [comment]
You're trying to use Debian repo on Ubuntu.

https://launchpad.net/~ondrej/+archive/ubuntu/php

**Reply by G400** [reply]
Thank you!
I managed to locate list where this was referenced now.
Problem solved

**Reply by Blackxes** [reply]
Can you please share where you found it?
I and maybe others encounter the same issue

Edit: Found it. In my case it was referenced in the `/etc/apt/sources.list.d/sury-php.list`
